models/repo/pushmirror.go

```diff
@@ -93,6 +94,15 @@ func GetPushMirrorsByRepoID(repoID int64) ([]*PushMirror, error) {
 	return mirrors, db.GetEngine(db.DefaultContext).Where("repo_id=?", repoID).Find(&mirrors)
 }
 
+// GetPushMirrorsByRepoIDWithSyncOnCommit returns push-mirror information of a repository,
+// filtered by sync_on_commit.
+func GetPushMirrorsByRepoIDWithSyncOnCommit(repoID int64, syncOnCommit bool) ([]*PushMirror, error) {
```

Comment:
What about
```suggestion
func GetPushMirrorsSyncedOnCommit(repoID int64, syncOnCommit bool) ([]*PushMirror, error) {
```
instead?
The current method name is way too long for me...

Also, your only use of this method seems to be for `syncOnCommit = true`, so you can delete this parameter as well shortening it to
```suggestion
func GetPushMirrorsSyncedOnCommit(repoID int64) ([]*PushMirror, error) {
```